k8s/openebs-operator.yaml

```diff
@@ -512,8 +509,7 @@ spec:
           exec:
             command:
             - pgrep
-            - -f
-            - ".*ndm"
+            - "^ndm.*"
```

Comment:
do we need regular expression here also?

Reply:
command name is only `ndm` , so i think we don't need * at the end